### DISCUSSION TITLE
CarePlan

### DISCUSSION THREAD

**Discussion by dt-r** [hl7au/au-fhir-core#13] [AU Core R1 Resource Priority List - CLOSED]
Upvote for inclusion of a Resource Profile of [CarePlan](http://hl7.org/fhir/R4/careplan.html) Resource in AU Core Release 1.

Comment if you think we need a different profile to AU Core CarePlan, or if there is a need for an additional CarePlan profile in AU Core.

AU Core CarePlan profile would set minimum expectations for a CarePlan resource to record, search, and fetch information about a care plan associated with a patient. 

AU Core CarePlan should be based on the [AU Base CarePlan](http://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-careplan.html) profile and identify the additional mandatory core elements, extensions, vocabularies and value sets that SHALL be present in the CarePlan resource when conforming to this profile. It provides the floor for standards development for specific uses cases in an Australian context.

**Comment by jackieoc370** [comment]
Types of care plans commonly raised by allied health professionals as needing to be included as priority are typically: NDIS plans, Behaviour support plans, chronic disease management plans (Medicare document). Others known as required to include are sick day plans for those with Type 1 diabetes
